example/lib/device_details/device_details_bloc.dart

```diff
@@ -112,6 +113,14 @@ class DeviceDetailsBloc {
         })
         .then((characteristics) => characteristics.forEach((characteristic) =>
             log("Found characteristic \n ${characteristic.uuid}")))
+        .then((_) async {
+            int rssi = await peripheral.rssi();
+            log("rssi $rssi");
+        })
+        .then((_) async {
+          await peripheral.requestMtu(74);
```

Comment:
We should log negotiated MTU

Reply:
will do in another PR

Reply:
Then create an issue for this on the board